frontend/src/common/components/OwnershipsList.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+
+import {Button, IconAlertCircleFill, IconCrossCircle, IconPlus} from "hds-react";
+import {useImmer} from "use-immer";
+import {v4 as uuidv4} from "uuid";
+
+import {useGetOwnersQuery} from "../../app/services";
+import {IOwner, IOwnership} from "../models";
+import {dotted, formatOwner} from "../utils";
+import {FormInputField} from "./index";
+
+const OwnershipsList = ({apartment}) => {
```

Comment:
What's this new component for? It's not used anywhere and seems a bit out of scope for this Ticket/PR (Not mentioned in the description)

Reply:
Yup, accidentally included in the PR 😅